tests/ui/eq_op.stderr

```diff
@@ -116,6 +116,19 @@ error: equal expressions as operands to `!=`
 LL |     ([1] != [1]);
    |     ^^^^^^^^^^^^
 
+error: strict comparison of f32 or f64
+  --> $DIR/eq_op.rs:25:5
+   |
+LL |     ([1] != [1]);
```

Comment:
These are integers and shouldn't trigger the lint. See comment above, how to fix this.